examples/pretrain_gpt_multilingual.sh

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash
+
+# Runs the "345M" parameter model
+
+GPUS_PER_NODE=8
+MASTER_ADDR=localhost
+MASTER_PORT=6000
+NNODES=1  # Adjust
+NODE_RANK=0  # Adjust
+WORLD_SIZE=$(($GPUS_PER_NODE*$NNODES))
+
+DISTRIBUTED_ARGS="--nproc_per_node $GPUS_PER_NODE --nnodes $NNODES --node_rank $NODE_RANK --master_addr $MASTER_ADDR --master_port $MASTER_PORT"
+
+# paths to multilingual preprocessed datasets
+DATA_PATH_EN=<Specify path and file prefix>_text_document
```

Comment:
which of these should be which of our resources or does order not matter?

Reply:
This is just an example that I took from the bigscience repo. The order does not matter

Reply:
In our case I think it'll be best to load the weights/splits/paths arguments from a file https://github.com/bigcode-project/Megatron-LM/blob/multi-validation/megatron/arguments.py#L995
I'll upload that soon